src/components/Spacer/Spacer.tsx

```diff
@@ -3,21 +3,34 @@ import React, { FunctionComponent, ReactNode } from 'react';
 import classNames from 'classnames';
 
 export interface SpacerProps {
-	side?: 'left' | 'top' | 'right' | 'bottom';
-	size?: 'l' | 's';
+	margin?: (
```

Comment:
laat mss ook toe dat we een single string doorgeven aangezien die dubbele opties niet vaak gaan voorkomen.
```
<Spacer margin={['left-small']}>
<Spacer margin="left-small">
```

Je kan de lijst met opties eerst definieren als volgt bovenaan de file:
```
type MarginOption = 'small'
		| 'large'
		| 'left-small'
		| 'left'
		| 'left-large'
		| 'top-small'
		| 'top'
		| 'top-large'
		| 'right-small'
		| 'right'
		| 'right-large'
		| 'bottom-small'
		| 'bottom'
		| 'bottom-large';
```

en dan wordt de prop type:
```
margin?: MarginOption[] | MarginOption;
```

en dan check je of het een string is en anders is het een array:
```
typeof margin === 'string'
```